__tests__/components/Comments/WriteComment.test.js

```diff
@@ -124,7 +124,7 @@ describe('WriteComment should run correctly', () => {
       backgroundColor: COLORS.balance_gray,
       position: 'absolute',
       top: 0,
-      left: 30,
+      left: 37,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the left position has been increased from 30 to 37. Again, please verify that this change doesn't cause any UI issues on different screen sizes.
```diff
- left: 30,
+ left: 37,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->